docs/extras/bootstrap.md

```diff
@@ -59,13 +61,13 @@ The `bootstrap_nav.*` templates produce the same output, and can be used as an e
 
 === `pagy_bootstrap_nav_js(pagy, ...)`
 
-![bootstrap_combo_nav_js](/docs/assets/images/bootstrap_nav_js-g.png)
+![bootstrap_nav_js - Notice how it gets shorter, depending on width?](/docs/assets/images/bootstrap_nav_js-g.png)
```

Comment:
It's just me, but could we just write something without questions like "bootstrap_nav_js - It can responsively adjust its :size to the available width"?

Reply:
> we should use the white background version.

Shall i get rid of the grey versions? I don't think we need them anymore (?)

Reply:
> Shall i get rid of the grey versions?

Yes please. I thought you were doing exactly that already. I completely forgot about the background when I worked on the new doc.

Reply:
My pics are white anyway (or should be close enough to white). I'll change the name of the files to reflect this to avoid confusion. apologies for it.

Reply:
we will have only one background so no need to rename and adding the abbreviation in the name.

Reply:
> It's just me, but could we just write something without questions like "bootstrap_nav_js - It can responsively adjust its :size to the available width"?
> 
> Also: we should use the white background version.

fixed with: [92ece79b](https://github.com/ddnexus/pagy/pull/453/commits/92ece79b6769d19a80bf77b4075425f470c46391)

Reply:
Re: Renaming of images:

fixed with [7aa46b96](https://github.com/ddnexus/pagy/pull/453/commits/7aa46b9664e9aa2db378b219aeaf9990356c1167)